index.bs

```diff
@@ -643,9 +643,12 @@ ReadableStream(<var>underlyingSource</var> = {}, { <var>size</var>, <var>highWat
          from _reader_.
        * If _reader_ is a <a>BYOB reader</a>, WritableStreamDefaultWriterGetDesiredSize(_writer_) should be used to
          determine the size of the chunks read from _reader_.
-       * Otherwise, WritableStreamDefaultWriterGetDesiredSize(_writer_) may be used to determine the flow rate
-         heuristically, e.g. by delaying reads while it is judged to be "low" compared to the size of chunks that have
-         been typically read.
+       * After taking into account these backpressure signals, reading and writing should be done promptly; other
```

Comment:
We can probably say less here now that we don't have to argue with the previous sentence. How about:

"Reads or writes should not be delayed for reasons other than backpressure."